code/modules/mob/living/carbon/xenomorph/castes/Warrior.dm

```diff
@@ -76,7 +78,8 @@
 
 
 /mob/living/carbon/Xenomorph/Warrior/stop_pulling()
```

Comment:
instead of adding a lunging var, just remove this stop_pulling thing entirely, modern woyers don't really need it